src/v2/Apps/Order/Components/PaymentPicker.tsx

```diff
@@ -520,6 +520,15 @@ export const PaymentPickerFragmentContainer = createFragmentContainer(
             country
             postalCode
           }
+          ... on CommerceShipArta {
+            name
+            addressLine1
+            addressLine2
+            city
+            region
+            country
+            postalCode
+          }
```

Comment:
I was initially unsure how adding fields to the query fragment would resolve this issue, but I think what is happening is the values that come back in the query [get added to the address that's sent to stripe](https://github.com/artsy/force/blob/8a1b817e2b85add7038e5776f0df57bbd4d53138/src/v2/Apps/Order/Components/PaymentPicker.tsx#L375-L380) -- do I understand that correctly?

Reply:
Fascinating find!

Reply:
yeah, that's pretty much it.